util/helper.ts

```diff
@@ -1,6 +1,36 @@
 import bcrypt from 'bcrypt';
-
+import { Request, Response } from 'express';
+import jwt from 'jsonwebtoken';
+import { AllowedAPIs } from '../contants/allowedApi.constant';
+import { unauthorized } from './response.helper';
+import { ResponseMessage } from '../contants/response-message.contant';
 
 export function comparePassword(password: string, encrptPassword: string): Promise<boolean> {
     return bcrypt.compare(password, encrptPassword);
-}
\ No newline at end of file
+}
+
+export function createToken(data: any): string {

```

Comment:
why any here? there should be a strict type here to pass data through which you will be going to create jwt token. So it would be great if you can create a type/interface or anything to place here the strict type for data to be included in the jwt token.

Reply:
updated with IAccount Model